preprocessing/shuffling/shuffle_labels.r

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env Rscript
+
+# Author_and_contribution: Niklas Mueller-Boetticher; created template
+# Author_and_contribution: Kim Vucinic; modified template and created script
+
+suppressPackageStartupMessages(library(optparse))
+
+# Arguments
+option_list <- list(
+  make_option(
+    c("-l", "--labels"),
+    type = "character", default = NULL,
+    help = "Labels from domain clustering. Path to labels (as tsv)."
+  ),
+  make_option(
+    c("--seed"),
+    type = "integer", default = NULL,
+    help = "Seed to use for random operations."
+  ),
+  make_option(
+    c("-o", "--out_file"),
+    type = "character", default = NULL,
+    help = "Output file."
+  )
+)
+
+# Description
+description <- "Shuffling labels..."
+
+opt_parser <- OptionParser(
+  usage = description,
+  option_list = option_list
+)
+opt <- parse_args(opt_parser)
+
+# Use these filepaths as input
+label_file <- opt$labels
+
+# Seed
+seed <- opt$seed
+set.seed(seed)
+
+## Your code goes here
+df <- read.delim(label_file, sep = "\t", row.names = 1)
+if (!("label" %in% colnames(df))){
+     stop("Label column not present in the file. Check your file.")
+}
+
+# Randomize labels
+df_randomized <- data.frame(label = sample(df$label))
```

Comment:
We might have an additional colum, in this dataframe that splits the label into high and low confidence. Should that be shuffled too?
@naveedishaque

Reply:
I think you mean "if we shuffle the `label` we should also shuffle the `confidence`"? 
My feeling is to keep a low confidence spot as a low confidence spot even after label shuffling. 
Does that make sense?

Reply:
Yeah, good point. In that case the code still needs to be adjusted to keep the additional columns untouched. Make sure only the labels are shuffled and the rownames still match all the other existing columns

Reply:
Sounds good. I'll add the changes :)